server/src/main/java/com/objectcomputing/checkins/services/agenda_item/AgendaItemController.java

```diff
@@ -40,105 +42,58 @@
-        return HttpResponse
-                .ok()
-                .headers(headers -> headers.location(
-                        URI.create(String.format("%s/%s", request.getPath(), updatedAgendaItem.getId()))))
-                .body(updatedAgendaItem);
+    @Post("/")
+    public HttpResponse<AgendaItem> createAgendaItem(@Body @Valid AgendaItemCreateDTO agendaItem, HttpRequest<AgendaItemCreateDTO> request) {
+        AgendaItem newAgendaItem = agendaItemServices.save(new AgendaItem(agendaItem.getCheckinid(), agendaItem.getCreatedbyid()
+                , agendaItem.getDescription()));
+        return HttpResponse.created(newAgendaItem)
+                .headers(headers -> headers.location(URI.create(String.format("%s/%s", request.getPath(), newAgendaItem.getId()))));
 
     }
 
     /**
-     * Delete agendaItem
+     * Update a agenda item
      *
-     * @param id, id of {@link AgendaItem} to delete
+     * @param agendaItem
+     * @param request
+     * @return
      */
-    @Delete("/{id}")
-    public HttpResponse<?> deleteAgendaItem(UUID id) {
-        agendaItemServices.delete(id);
-        return HttpResponse
-                .ok();
+    @Put("/")
+    public HttpResponse<AgendaItem> updateAgendaItem(@Body @Valid AgendaItem agendaItem, HttpRequest<AgendaItemCreateDTO> request) {
+        AgendaItem updateAgendaItem = agendaItemServices.update(agendaItem);
+        return HttpResponse.ok().headers(headers -> headers.location(
+                URI.create(String.format("%s/%s", request.getPath(), updateAgendaItem.getId()))))
+                .body(updateAgendaItem);
     }
 
     /**
-     * Get AgendaItem based off id
+     * Get items by checkind or createbyid
      *
-     * @param id {@link UUID} of the agenda item entry
-     * @return {@link AgendaItem}
-     */
-    @Get("/{id}")
-    public AgendaItem readAgendaItem(UUID id) {
-        return agendaItemServices.read(id);
-    }
-
-    /**
-     * Find agenda items that match all filled in parameters, return all results when given no params
-     *
-     * @param checkinid   {@link UUID} of checkin
-     * @param createdbyid {@link UUID} of member
-     * @return {@link List < CheckIn > list of checkins}
+     * @param checkinid
+     * @param createdbyid
+     * @return
      */
     @Get("/{?checkinid,createdbyid}")
-    public Set<AgendaItem> findAgendaItems(@Nullable UUID checkinid,
-                                           @Nullable UUID createdbyid) {
+    public Set<AgendaItem> findAgendaItem(@Nullable UUID checkinid,
+                                            @Nullable UUID createdbyid) {
         return agendaItemServices.findByFields(checkinid, createdbyid);
     }
 
     /**
-     * Load agenda items
+     * Get agenda item from id
      *
-     * @param agendaItems, {@link List< AgendaItemCreateDTO > to load {@link AgendaItem agenda items}}
-     * @return {@link HttpResponse<List< AgendaItem >}
+     * @param id
+     * @return
      */
-    @Post("/items")
```

Comment:
why was this end point deleted. ?

Reply:
thanks, added back, got removed somehow.